torchrl/envs/transforms/transforms.py

```diff
@@ -2429,3 +2429,50 @@ def __repr__(self) -> str:
             f"{self.__class__.__name__}(decay={self.decay:4.4f},"
             f"eps={self.eps:4.4f}, keys={self.in_keys})"
         )
+
+
+class StepCounter(Transform):
+    """Counts the steps from a reset and sets the done state to True after a certain number of steps.
+
+    Args:
+        max_steps (:obj:`int`, optional): the maximum number of steps to take before setting the done state to True.
```

Comment:
+ add what the default is, and if it is None, what behaviour is expected in that case.